postman_tests/config/test_env.json

```diff
@@ -0,0 +1,15 @@
+{
+	"id": "9dbcc6c6-dd15-498a-a4f7-5fc0cd63d542",
+	"name": "chatbot-ner-prod",
+	"values": [
+		{
+			"key": "url",
+			"value": "localhost:8081",
+			"enabled": true
+		}
+	],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2020-03-05T12:18:03.223Z",
+	"_postman_exported_using": "Postman/7.19.1",
+	"datastore_host": "localhost:8081"
```

Comment:
Minor, please don't call it `datastore_host`, call it simply `server_url` because it is in fact where uwsgi server is running